langs/in.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Indonesian",
+  "code": "es",
```

Comment:
I think the code is `in`

Reply:
yeah, sorry about that, fixed

Reply:
Well that was actually changed in 1989. The new code is `id`. CMIIW.

Reply:
Yeah, it should probably be `id`! Sorry.